PokemonRampUp/Model/WebServices.swift

```diff
@@ -21,22 +21,22 @@ class WebServices {
         return Int.random(in: 1...1008)
```

Comment:
Only made changes necessary to the `WebServices` class so that the app would compile. This file does not need to be reviewed.